pipelines/tekton/azureml-container-pipeline/kserve-download-model.yaml

```diff
@@ -12,13 +12,30 @@ spec:
     type: string
   - name: aws-bucket-name
     type: string
+  - name: fetch-github-model
+    type: string
+    default: "true"
+  - name: fetch-S3-model
+    type: string
+    default: "false"
+  - name: github-url
+    type: string
+    default: https://github.com/opendatahub-io/ai-edge/raw/3dd0242871371cf66af3f20e3a8924a4b5fc1658/pipelines/models/bike-rentals-auto-ml/model.pkl
   steps:
-  - name: download-model
+  - name: download-model-github
+    image: curlimages/curl:latest
```

Comment:
Please fully qualify the image name.

Reply:
I think we might no longer need this, but if this remains, I'll update with the full image. 👍🏻